.gitignore

```diff
@@ -2,6 +2,7 @@
 .DS_Store
 
 # NPM
+package-lock.json
```

Comment:
This may be a little controversial. My thinking way to omit the `package-lock.json` file until we normalize on Node 8 / NPM 5 across the team.

Reply:
Yeah, The Package-lock.json file will be Good For This.
Hopeefully, The NodeNpm Can Be Installted.